participant-manager/src/app/site-coordinator/studies/study-list/study-list.component.html

```diff
@@ -1,9 +1,14 @@
 <div class="max__w__1280">
```

Comment:
Should this have copyright header?

Reply:
Hi @jkrogers18,  we include copyright headers in only SVG images, not in HTML and CSS files.